src/settings/FinePolicy/components/EditSections/RangeSection/OverdueFinesSectionColumn.js

```diff
@@ -48,8 +48,7 @@ const OverdueFinesSectionColumn = (props) => {
                   { value: true, label: formatMessage({ id: 'ui-circulation.settings.finePolicy.yes' }) },
                   { value: false, label: formatMessage({ id: 'ui-circulation.settings.finePolicy.no' }) },
                 ]}
-              />
-            }
+              />}
```

Comment:
OMG this kind of change is a waste of our time. I have opposition to it now that it's done, but what is the point???